### ISSUE TITLE
Request for example: Weight Decay Optimizers / Super Convergence

### ISSUE THREAD

**Issue by seanpmorgan** [tensorflow/addons#336]
A valuable set of optimizers that are available in Addons.

Related info:
https://www.fast.ai/2018/07/02/adam-weight-decay/
https://github.com/tensorflow/addons/blob/master/examples/README.md

**Comment by SSaishruthi** [comment]
I can take up this one if it is ok. Is it ok if we build this example based on `lazy_adam` example?
@seanpmorgan

**Comment by seanpmorgan** [comment]
> I can take up this one if it is ok. Is it ok if we build this example based on `lazy_adam` example?
> @seanpmorgan

Yeah we can use that as a starting point. I would like to include some info from that Fast AI blog (credit them where appropriate)... and it would be great to show the convergence speed against standard adam optimizer

**Comment by SSaishruthi** [comment]
Sure. I will keep you posted about the progress.

**Comment by SSaishruthi** [comment]
@seanpmorgan Can you please assign this issue to me? I am working on it

**Comment by seanpmorgan** [comment]
TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)
